hazelcast-oss/Dockerfile

```diff
@@ -27,7 +27,7 @@ COPY *.jar get-hz-dist-zip.sh ${HZ_HOME}/
 
 # Install
 RUN echo "Installing new APK packages" \
-    && apk add --no-cache openjdk11-jre-headless bash curl libxml2-utils zip unzip \
+    && apk add --no-cache openjdk11-jre-headless bash curl libxml2-utils zip unzip curl \
```

Comment:
`curl` is already in this line, you only need to remove it from line: `&& apk del libxml2-utils curl zip unzip`

Reply:
Nice catch :+1: